lisa/sut_orchestrator/azure/features.py

```diff
@@ -182,6 +185,191 @@ def is_over_nd(self) -> bool:
         dmesg = self._node.tools[Dmesg]
         return "hvnd_try_bind_nic" in dmesg.get_output()
 
+    def setup_rdma(self) -> None:
```

Comment:
It looks like the logic is not Azure specified. It should be in the generic Feature code, not in the Azure.